packages/cli/src/commands/deploy/packing/nft.js

```diff
@@ -7,6 +7,8 @@ import fse from 'fs-extra'
 import { findApiDistFunctions } from '@redwoodjs/internal/dist/files'
 import { ensurePosixPath, getPaths } from '@redwoodjs/project-config'
 
+import * as nftPacker from '../packing/nft'
```

Comment:
Do we have any idea about what changed and broke this in the first place? This is a surprising fix given that the error showed up after a minor release... Assuming the culprit could be with the NFT package itself. 🤷‍♂️

No need to hold up the PR to try and figure it out — goal here is simple + efficient. Just thought it was worth asking.

Reply:
I am not 100% sure - I got to this fix by simple trial and error, realizing eventually that "exports" is undefined. Having a look around the web now , it has to do with how the JS is compiled / emitted. Modern versions don't allow the `exports.xxx` syntax depending on module resolution.

I am not familiar enough with the RW codebase to see what change might have caused this. I did go through our build log and can confirm that in `@redwoodjs/cli@npm:7.2.0-rc.53` this still worked and packed correctly.

Reply:
> it has to do with how the JS is compiled / emitted

Yeah, this is the kind of thing that has me thinking a bit; one of our related build dependencies could have been updated behind the scenes and introduced a behavior change that's breaking. 

Again, not a critical priority to figure out. Appreciate you taking time to reply 👍